dymaptic.Pro.ZoomToCoordinates/Views/About.xaml.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Diagnostics;
+using System.Windows.Navigation;
+
+namespace dymaptic.Pro.ZoomToCoordinates.Views
+{
+    /// <summary>
+    /// Interaction logic for About.xaml
+    /// </summary>
+    public partial class About : ArcGIS.Desktop.Framework.Controls.ProWindow
+    {
+        public About()
+        {
+            InitializeComponent();
+        }
+
+        private void Hyperlink_RequestNavigate(object sender, RequestNavigateEventArgs e)
+        {
+            var psi = new ProcessStartInfo
+            {
+                FileName = e.Uri.AbsoluteUri,
+                UseShellExecute = true
+            };
+            Process.Start(psi);
```

Comment:
cool, this launches the default web browser, I assume?

Reply:
Yup, default web browser.